src/invariant/test/BucketListIsConsistentWithDatabaseTests.cpp

```diff
@@ -48,6 +48,20 @@ struct BucketListGenerator
         key.account().accountID = skey.getPublicKey();
         mLiveKeys.insert(key);
 
+#ifdef ENABLE_NEXT_PROTOCOL_VERSION_UNSAFE_FOR_PRODUCTION
+        // All config settings entries will be created automatically during
```

Comment:
Does that mean we can remove the exclusions from other tests?

Reply:
Which other tests? The exclusions in BucketListIsConsistentWithDatabaseTests are still required?